Resolve panic in production caused by cache config

Recently the bot moved to using disgord and initially used a cache config disabling all caches within the bot, as it only needs to respond to messages on-demand without anything being held in the cache.

However, in the production environment, this cache config resulted in the bot panicking and exiting. Commenting out the cache config resolve this for now but results in more memory usage for the bot.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x7789d7]

goroutine 52 [running]:
github.com/andersfylling/disgord.(*Cache).UpdateMemberAndUser(0xc000118330, 0x530f8633f820003, 0x8e2734314800029, 0xc0008062c0, 0x14f, 0x160)
        C:/Users/Administrator/go/pkg/mod/github.com/andersfylling/disgord@v0.17.3/cache.go:783 +0x57
github.com/andersfylling/disgord.cacheEvent(0x9a71e0, 0xc000118330, 0xc0001503e0, 0x13, 0x879ae0, 0xc0000da230, 0xc0008062c0, 0x14f, 0x160, 0x0, ...)
        C:/Users/Administrator/go/pkg/mod/github.com/andersfylling/disgord@v0.17.3/events.go:109 +0x184c
github.com/andersfylling/disgord.demultiplexer(0xc0000d2940, 0xc000108420, 0xc000118330)
        C:/Users/Administrator/go/pkg/mod/github.com/andersfylling/disgord@v0.17.3/reactor.go:73 +0x207
created by github.com/andersfylling/disgord.(*Client).setupConnectEnv
        C:/Users/Administrator/go/pkg/mod/github.com/andersfylling/disgord@v0.17.3/client.go:393 +0x216
```

(_Yes, production is a Windows server environment, don't @ me_)

--- a/go.mod
+++ b/go.mod
@@ -3,7 +3,7 @@ module github.com/MattIPv4/1.1.1.1-Discord
 go 1.14
 
 require (
-	github.com/andersfylling/disgord v0.17.3
+	github.com/andersfylling/disgord v0.17.4-0.20200616052813-1d950740462f
 	github.com/jakemakesstuff/structuredhttp v0.0.0-20200426130406-5e3dab895f6c
 	github.com/olekukonko/tablewriter v0.0.4
 )

--- a/go.mod
+++ b/go.mod
@@ -3,7 +3,7 @@ module github.com/MattIPv4/1.1.1.1-Discord
 go 1.14
 
 require (
-	github.com/andersfylling/disgord v0.17.4-0.20200616052813-1d950740462f
+	github.com/andersfylling/disgord v0.18.0
 	github.com/jakemakesstuff/structuredhttp v0.0.0-20200426130406-5e3dab895f6c
 	github.com/olekukonko/tablewriter v0.0.4
 )

--- a/main.go
+++ b/main.go
@@ -68,12 +68,12 @@ func main() {
 	// Create a new Discord session using the provided bot token.
 	dg := disgord.New(disgord.Config{
 		BotToken: Token,
-		/*CacheConfig: &disgord.CacheConfig{
+		CacheConfig: &disgord.CacheConfig{
 			DisableChannelCaching:    true,
 			DisableUserCaching:       true,
 			DisableVoiceStateCaching: true,
 			DisableGuildCaching:      true,
-		},*/
+		},
 	})
 
 	// Register the events to set the user ID and manage counts.

